### PR TITLE
🔥🔥🔥migrated from ava to jest. Added shim spec 🔥🔥🔥

### PR DESCRIPTION
**Pull request by mergebandit**
FYI - had to include a resolution for babel-core

https://github.com/facebook/jest/issues/6913

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,11 @@
     "prebuild": "yarn run clean",
     "build": "babel src -d dist",
     "clean": "del dist",
-    "test": "eslint ."
+    "lint": "eslint .",
+    "test": "yarn run lint && jest"
+  },
+  "resolutions": {
+    "babel-core": "7.0.0-bridge.0"
```
Comment (transitive-bullshit) [inline package.json]: link to a guide on how this works with the 6->7 transition?

File: package.json [hunk 2]
```diff
@@ -35,12 +39,22 @@
     "@babel/plugin-proposal-class-properties": "^7.1.0",
     "@babel/preset-env": "^7.1.0",
     "ava": "^0.25.0",
+    "babel-eslint": "^10.0.1",
+    "babel-jest": "^23.6.0",
+    "babel-preset-env": "^1.7.0",
```
Comment (transitive-bullshit) [inline package.json]: all of this is the old babel; the new stuff is @babel/* as you can see on line 40 here

File: test/__snapshots__/shim.spec.js.snap [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```
Comment (transitive-bullshit) [inline test/__snapshots__/shim.spec.js.snap]: sooooo sexy

File: test/fixtures/basic.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { storiesOf } from '@storybook/react'
+import { storiesOf } from '../../src/storybook-react-client-shim'
```
Comment (transitive-bullshit) [inline test/fixtures/basic.js]: pretty sure you wanna rebase cause this file's now called just 'shim.js'

File: test/shim.spec.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import renderer from 'react-test-renderer'
+import { getStorybook } from '../src/storybook-react-client-shim'
+
+describe('Storybook Shim', () => {
+  describe('against Basic Fixture', () => {
+    let ButtonStory, story1, story2
+
+    beforeAll(() => {
+      require('./fixtures/basic')
+      ;[ButtonStory] = getStorybook()
```
Comment (transitive-bullshit) [inline test/shim.spec.js]: booo semicolons; make these separate statements (nit)

Comment (hydratedev) [reply test/shim.spec.js]: Boo this: 
```
const storyBook = getStorybook()
ButtonStory = storyBook[0]
story1 = ButtonStory.stories[0]
story2 = ButtonStory.stories[1]
```

File: package.json (hunk 1 above)
Comment (mergebandit) [reply package.json]: (in description as well: https://github.com/facebook/jest/issues/6913)

**Pull request merged**